setup.py

```diff
@@ -144,6 +160,6 @@ def write_version_py(filename=None):
       install_requires=INSTALL_REQUIRES,
       tests_require=TESTS_REQUIRE,
       url=URL,
-      test_suite='nose.collector',
+      cmdclass={'test': PyTest},
```

Comment:
No we don't need it.  I also never use `setuptools` to run my test package.  I included it because we previously had the `nose.collector` line in there that provided that functionality.